pages/about/contributing-in-github.md

```diff
@@ -0,0 +1,119 @@
+---
+title: Contributing in GitHub
+search_exclude: true
+description: Instructions on how to contribute content to the DS Handbook in GitHub
+contributors: Marina Popleteeva, Diana Pilvar, Niclas Jareborg, Helena SchnitSchnitzer, Jeanne Wilbrandt
```

Comment:
Helena Schnitzer. Also, my name is missing :)

Reply:
Fixed